api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8946,4 +8946,61 @@
             <column name="address15" type="varchar(255)"></column>
         </addColumn>
     </changeSet>
+
+
+    <changeSet id="TRUNK-3422-adding-View-Visits-Privilege" author="PralayRamteke" >
+            <preConditions  onFail="MARK_RAN">
+                <not>
+                    <sqlCheck expectedResult="0">
+                    SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
+                    </sqlCheck>
+                </not>
+            </preConditions>
+            <comment>
+                Add "View Visits" privilege to the roles having "View Encounter"
+            </comment>
+                <sql>
+                    INSERT INTO privilege (privilege,description)
```

Comment:
@dkayiwa suppose View Visit does not exist in privilege table(this implies that View Visits does not exists in role_privilege table) and "View Encounters" Exist in role_privilege i.e. some role has "View Encounters" privilege then this code first adds "View Visits" in privilege tables if it already does not exist and then it add "View Visits" privilege to the role which has 'View Encounter " privilege if this role does not have "View Visits" already!